internal/dic/dic.go

```diff
@@ -117,14 +127,17 @@ func (d *Dic) loadUnkDicPart(r io.Reader) error {
 }
 
 // Load loads a dictionary from a file.
-func Load(path string) (d *Dic, err error) {
-	d = new(Dic)
+func Load(path string, simple bool) (d *Dic, err error) {
 	r, err := zip.OpenReader(path)
 	if err != nil {
 		return d, err
 	}
 	defer r.Close()
+	return load(&r.Reader, simple)
+}
 
+func load(r *zip.Reader, full bool) (d *Dic, err error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 20 of function load() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->